src/components/Accordion/AccordionHeader.tsx

```diff
@@ -4,25 +4,25 @@ import React, { ReactElement, cloneElement, isValidElement, useState } from 'rea
 import { IconSize } from '@foundation/Icon/IconSize';
 import { IconProps } from '@foundation/Icon/IconProps';
 import { merge } from '@utilities/merge';
-import { AccordionHeaderIconSize, AccordionHeaderProps } from './types';
+import { AccordionHeaderProps } from './types';
 import { AccordionHeaderIcon } from './AccordionHeaderIcon';
 
+export const ACCORDION_HEADER_TEST_ID = 'fondue-accordion-header';
+
 export const AccordionHeader = ({
     isOpen = false,
     decorator,
     disabled = false,
     children,
-    size = AccordionHeaderIconSize.Medium,
-    bold = true,
-    type,
+    'data-test-id': testId = ACCORDION_HEADER_TEST_ID,
```

Comment:
I think it's better to use the same name everywhere and just name it `dataTestId`

Reply:
Perfect. Will change it and we keep it consistent